hyperliquid/exchange.py

```diff
@@ -49,25 +49,28 @@ def _post_action(self, action, signature, nonce):
         return self.post("/exchange", payload)
 
     def order(
-        self, coin: str, is_buy: bool, sz: float, limit_px: float, order_type: OrderType, reduce_only: bool = False
+        self, coin: str, is_buy: bool, orders: List[Dict[str, float]], order_type: OrderType, reduce_only: bool = False
```

Comment:
Also, it would be great to make the types more specific than List[Dict[str, float]]. Ideally, end users would get a type error if they forget a required field or pass in a field that isn't supported.